mesh-worker-service/src/main/java/io/functionmesh/compute/rest/api/FunctionsImpl.java

```diff
@@ -69,17 +69,32 @@ private void validateRegisterFunctionRequestParams(String tenant, String namespa
         if (functionConfig == null) {
             throw new RestException(Response.Status.BAD_REQUEST, "Function config is not provided");
         }
+        Map<String, Object> customConfig = worker().getWorkerConfig().getRuntimeCustomizerConfig();
+        if (jarUploaded &&  customConfig != null && customConfig.get("uploadEnabled") != null &&
+                ! (Boolean) customConfig.get("uploadEnabled") ) {
+            throw new RestException(Response.Status.BAD_REQUEST, "Uploading Jar File is not enabled");
+        }
     }
 
     private void validateUpdateFunctionRequestParams(String tenant, String namespace, String functionName,
-                                                     FunctionConfig functionConfig) {
-        validateRegisterFunctionRequestParams(tenant, namespace, functionName, functionConfig);
+                                                     FunctionConfig functionConfig, boolean uploadedJar) {
+        validateRegisterFunctionRequestParams(tenant, namespace, functionName, functionConfig, uploadedJar);
     }
 
     private void validateGetFunctionInfoRequestParams(String tenant, String namespace, String functionName) {
         this.validateGetInfoRequestParams(tenant, namespace, functionName, kind);
     }
 
+    private void validateFunctionEnabled() {
+        Map<String, Object> customConfig = worker().getWorkerConfig().getFunctionsWorkerServiceCustomConfigs();
+        if (customConfig != null) {
+            Boolean functionEnabled = (Boolean) worker().getWorkerConfig().getRuntimeCustomizerConfig().get("functionEnabled");
```

Comment:
we can reuse `customConfig` to get the config.
```suggestion
            Boolean functionEnabled = (Boolean) customConfig.get("functionEnabled");
```